backend/app/views/spree/admin/trackers/index.html.erb

```diff
@@ -15,15 +15,13 @@
 <% if @trackers.any? %>
   <table class="index">
     <colgroup>
-      <col style="width: 30%">
-      <col style="width: 40%">
-      <col style="width: 15%">
-      <col style="width: 15%">
+      <col style="width: 35%">
+      <col style="width: 45%">
+      <col style="width: 20%">
```

Comment:
This was not in the original work either. Since a column was being removed, I updated the colgroup respectively.